exercicios/para-casa/entregas/ana-carolina-zago/contaBancaria.js

```diff
@@ -0,0 +1,39 @@
+class contaBancaria {
+    constructor(titular, saldo, limite) {
+        this.titular = titular;
+        this.saldo = saldo;
+        this.limite = limite;
+    }
+
+    // consulta de saldo
+    consultarSaldo() {
+        return this.saldo
+    }
+
+    // consulta de limite
+    consultarLimite() {
+        return this.limite
+    }
+
+    // consulta de saque
+    realizarSaque(valor) {
+        if(valor <= this.saldo && valor <= this.limite){
+            return "Saque realizado com sucesso"
+        }else if(valor > this.saldo || valor <= this.limite) {
+            return "Saque realizado com sucesso"
```

Comment:
aqui seria bom retornar também o novo valor do saldo, que no caso sendo maior que a quantia disponível, retornaria um valor negativo.
ex: return this.saldo = this.saldo - valor (ou também naquela versão refatorada usando -= valor)

Reply:
Ótima contribuição Melina! Vou fazer isso!!!